spec.bs

```diff
@@ -1356,6 +1356,17 @@ To <dfn>parse an https origin</dfn> given a [=string=] |input|:
 <div algorithm>
 
 To <dfn>build bid generators map</dfn> given an [=auction config=] |auctionConfig|:
+
+  <div class="note"> This algorithm builds an [=ordered map=] with:
+    * key: An [=origin=], which is a buyer of [=auction config/interest group buyers=]

```

Comment:
May this be easier to read with a tuple key? Though I guess how we only care about joining origin some of the time may make it messy